feature-assets/src/main/java/io/novafoundation/nova/feature_assets/presentation/transaction/history/mixin/OperationMappers.kt

```diff
@@ -84,6 +82,15 @@ private fun transferDirectionIcon(isIncome: Boolean): Int {
     return if (isIncome) R.drawable.ic_arrow_down else R.drawable.ic_arrow_up
 }
 
+@ColorRes
+private fun incomeTextColor(isIncome: Boolean, operationStatus: Operation.Status): Int {
+    return when {
+        operationStatus == Operation.Status.FAILED -> R.color.text_tertiary
```

Comment:
I checked our design system and I didn't find text_tertiary there, only text_secondary with the same color. Already asked a designer

Reply:
Afair it was reverted by Ruslan's request. I will re-check with mockups and update it if needed